debian/changelog

```diff
@@ -0,0 +1,9 @@
+authd (0.1) noble; urgency=medium
```

Comment:
Mh, not sure it's the best behavior IMHO, mostly because you may want to compile this also for older revisions if you need to backport. So I'd use `UNRELEASED` until a real release happens.

Reply:
Yeah, still you know... It's like a "stamp" on the package, so normally marking it as released and signing the source is something that should come together.

See also: https://wiki.ubuntu.com/DesktopTeam/git#Release_a_package